A-git-in-other-environments.asc

```diff
@@ -13,13 +13,14 @@ include::book/A-git-in-other-environments/sections/visualstudio.asc[]
 
 include::book/A-git-in-other-environments/sections/eclipse.asc[]
 
-
 include::book/A-git-in-other-environments/sections/bash.asc[]
 
 include::book/A-git-in-other-environments/sections/zsh.asc[]
 
 include::book/A-git-in-other-environments/sections/powershell.asc[]
 
+included::book/A-git-in-other-environments/sections/sublimetext.asc[]
+
```

Comment:
Can you move this up to just below the Eclipse section?

Reply:
Done.